src/Widgets/EventRow.vala

```diff
@@ -20,20 +20,25 @@
 public class DateTime.EventRow : Gtk.ListBoxRow {
     public GLib.DateTime date { get; construct; }
     public unowned ICal.Component component { get; construct; }
+    public unowned E.SourceCalendar cal { get; construct; }
 
     public GLib.DateTime start_time { get; private set; }
     public GLib.DateTime? end_time { get; private set; }
     public bool is_allday { get; private set; default = false; }
 
     private static Services.TimeManager time_manager;
     private static Gtk.CssProvider css_provider;
+    private static Gtk.CssProvider css_color_provider;
```

Comment:
This should not be declared  `static` - it is assigned in the instance construct clause.